nextflow.config

```diff
@@ -35,13 +35,17 @@ profiles {
   // Same as O2massive but with GPU support for selected processes
   O2gpu {
     includeConfig 'config/O2base.config'
-    includeConfig 'config/massive.config'
+    includeConfig 'config/gpu.config'
     
     process {
```

Comment:
@clarenceyapp I suggest moving these `process {...}` directives to `config/gpu.config` that you just created.